modules/trusted-firmware-m/Kconfig.tfm

```
@@ -20,6 +20,8 @@ config TFM_BOARD
 	default "stm/b_u585i_iot02a" if BOARD_B_U585I_IOT02A
 	default "stm/nucleo_l552ze_q" if BOARD_NUCLEO_L552ZE_Q
 	default "stm/stm32l562e_dk" if BOARD_STM32L562E_DK
+	default "stm/stm32wba65i_dk" if BOARD_NUCLEO_WBA65RI
+	default "stm/stm32wba65i_dk" if BOARD_STM32WBA65I_DK1
```

Maybe:
```suggestion
	default "stm/stm32wba65i_dk" if BOARD_NUCLEO_WBA65RI || BOARD_STM32WBA65I_DK1
```

But is sharing the same value for different boards expected?